kart/geometry.py

```diff
@@ -223,6 +223,10 @@ def from_hex_wkb(cls, wkb):
     def from_hex_ewkb(cls, hex_ewkb):
         return hex_ewkb_to_gpkg_geom(hex_ewkb)
 
+    @classmethod
+    def from_bbox(cls, min_x, max_x, min_y, max_y):
```

Comment:
actually OGR uses this ordering in at least one place too (`OGRLayer.GetExtent()`). So I guess this is a reasonably widely used format. Whatever's easiest 🤷

Reply:
This is by now - for better or worse - the closest thing we have to a standard Kart-internal ordering. This is the order that GPKG envelopes use - either `(-x, +x, -y, +y)` or `(-x, +x, -y, +y, -z, +z)` and kart features are stored as GPKG geometries with envelopes. So, I'd argue that this nothing we can really gain by moving toward a more _conventional_ choice, given we'd be moving further away from both PDAL and GPKG and so making Kart more inconsistent.
Happily, it's mostly not important for users since we don't generally expose envelopes.

A notable exception is our spatial-filter-extension syntax which uses: `(w, s, e, n)`